src/placeos-models/user.cr

```diff
@@ -164,7 +164,11 @@ module PlaceOS::Model
     secondary_index :email
 
     def self.find_by_email(authority_id : String, email : String)
-      User.where(email: email, authority_id: authority_id).first?
+      User.collection_query { |table|
```

Comment:
since `authority_id` is a secondary index, you can use `get_all` here instead of filter

Reply:
nice, will do